src/utils.js

```diff
@@ -197,11 +200,32 @@ export const PopperPlacements = [
   'bottom-start',
   'left-end',
   'left',
-  'left-start',
+  'left-start'
 ];
 
 export const canUseDOM = !!(
   typeof window !== 'undefined' &&
   window.document &&
   window.document.createElement
 );
+
+export function findDOMElements(el) {
```

Comment:
This is similar to `getTarget` above with the differences being this only accepts a string to pass to `querySelectorAll`, it returns a list of elements, and it more SSR save.
Can this be refactored to allow for different types of arguments (such as what `getTarget` allows: function, string, raw DOM reference). Then can `getTarget` be refactored to use this can just grab the first element in the list.